packages/blade/src/components/Badge/StyledBadge.native.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components/native';
+import { getStyledBadgeStyles } from './getStyledBadgeStyles';
+import type { StyledBadgeProps } from './StyledBadge.d';
+
+const StyledBadge = styled.View<StyledBadgeProps>((props) => ({
+  ...getStyledBadgeStyles(props),
+  alignSelf: 'center',
```

Comment:
Why is this `center` and not `flex-start` 🤔

Reply:
Same discussion like the one we had for IconButton. Here centering it makes even more sense since when a badge is placed in a row in a table, the most usual use-case is to center align it.

Reply:
On web, it is aligned to start 😀 
<img width="226" alt="image" src="https://user-images.githubusercontent.com/6682655/185044800-600a2ed4-55e0-4227-9972-ef1032675ecc.png">